kyuubi-common/src/main/scala/org/apache/kyuubi/KyuubiSQLException.scala

```diff
@@ -47,22 +60,24 @@ object KyuubiSQLException {
   private final val HEAD_MARK: String = "*"
   private final val SEPARATOR: Char = ':'
 
-  def apply(msg: String, throwable: Throwable): KyuubiSQLException = {
-    new KyuubiSQLException(msg, findCause(throwable))
+  def apply(
+      msg: String,
+      cause: Throwable = null,
+      sqlState: String = null,
+      vendorCode: Int = 0): KyuubiSQLException = {
```

Comment:
`vendorCode: Int = 0`

Why we use `0` as the default code, seems `0` is success. use `-1` ?

Reply:
nvm, `SQLException` also use 0 ..